pgoapi/pgoapi.py

```diff
@@ -224,6 +229,16 @@ def heartbeat(self):
             self.attempt_evolve(self.inventory.inventory_items)
             self.cleanup_pokemon(self.inventory.inventory_items)
 
+        # Auto-use lucky-egg if applicable
+        if self.auto_use_lucky_egg and self.inventory.has_lucky_egg() and current_time - self._last_egg_use_time > 30*60:
+            self.use_item_xp_boost(item_id=Inventory.ITEM_LUCKY_EGG)
+            response = self.call()
+
+            if 'USE_ITEM_XP_BOOST' in response['responses'] and 'result' in response['responses']['USE_ITEM_XP_BOOST'] and response['responses']['USE_ITEM_XP_BOOST']['result'] == 1:
```

Comment:
Good question! I actually tried to see if I could get any other response code with other scenarios (for instance, if the lucky egg is already in use), and I don't even get the 'result' property back, hence why I have to check it in the conditional :(